acceptance/tests/provider/package/pip.rb

```diff
@@ -22,21 +26,42 @@
       end
     end
 
-    step "Install a pip package" do
+    step "Ensure presence of a pip package" do
       package_manifest = resource_manifest('package', package, { ensure: 'present', provider: 'pip' } )
       apply_manifest_on(agent, package_manifest, :catch_failures => true) do
         list = on(agent, "#{pip_command} list --disable-pip-version-check").stdout
         assert_match(/#{package} \(/, list)
       end
-      on(agent, "#{pip_command} uninstall #{package} --disable-pip-version-check --yes")
```

Comment:
Why have you changed this? I find the behavior that a step removes a package it installed fair enough

Reply:
I initially thought that cleaning up after doing all the steps, through the added `teardown`, would be enough and I saw that line useless for the current combination of steps. 
But now that you've pointed it out, I agree that it would be nice to leave a clean environment after every step. It could get frustrating for someone who will want to modify or add new steps when random failures would occur.
Added the uninstall back 😁.